torch/csrc/jit/passes/symbolic_shape_runtime_fusion.cpp

```diff
@@ -309,49 +470,70 @@ RegisterOperators reg_guard({
             // each invocation or would that mess up with multithreaded
             // inference since we are writing to it?
             // TODO - smallvector here ?
+
             std::vector<int64_t> flattened_symbolic_dims(num_symbolic_dims, -1);
             size_t flattened_dim_offset = 0;
+            size_t flattened_stride_offset = 0;
             for (const auto i : c10::irange(num_inputs)) {
               at::Tensor tensor = inputs[i].toTensor();
               if (C10_UNLIKELY(
                       tensor.device() != device ||
                       tensor.dtype() != expected_scalar_types[i]) ||
                   tensor.requires_grad()) {
-                std::cout << " DEVICE /DTYPE/ REQUIRES GRAD MISMATCH\n";
```

Comment:
I'm wary of runtime VLOG's, because this will get executed on every execution. Even if debug is set to false, we have to make a call to look up the environment flag which is slow